src/app/(pre-login)/logging-in/page.tsx

```diff
@@ -22,7 +22,7 @@ import { useAuthContext } from '@/global/utils/auth';
 import { DnaLoader, css, useTheme } from '@icgc-argo/uikit';
 import { useRouter } from 'next/navigation';
 import { useQuery } from 'react-query';
-import { useAppConfigContext } from '../components/ConfigProvider';
+import { useAppConfigContext } from '../../components/ConfigProvider';
```

Comment:
these could all more easily be handled with absolute paths. no need to update them as often that way.
e.g. `@/components...`